src/ngamsServer/ngamsServer/ngamsServer.py

```diff
@@ -2392,7 +2391,7 @@ def _parse_and_run(args, prog, server_class):
         larg = arg.lower()
         if larg in _lower:
             if larg != arg:
-                print("WARNING: case-insenstive command-line option names are "
+                print("WARNING: case insensitive command-line option names are "
```

Comment:
The two changes in this file seem unrelated to this PR, could they be removed (hopefully through a rebase to leave no trace in the git history, otherwise I'll squash the commits when I merge)? I was expecting no changes in `ngmasServer.py` and these break that expectation.

Reply:
I small typo fix. It somehow slipped in with these changes. I am unsure how to undo it.

Reply:
OK, no problems, let's just leave it as is, it was a small detail anyway